Resources/translations/time.nl.xliff

```diff
@@ -56,4 +56,4 @@
             </trans-unit>
         </body>
     </file>
-</xliff>
+</xliff>
```

Comment:
Was just correcting the conflict that appeared during rebase, no idea why it did. But those two files are the same as you can see.